app/serializers/user_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class UserSerializer < ActiveModel::Serializer
+  attributes :username, :email, :api_token
```

Comment:
Be careful with this since it's the default serializer for a user and you don't want to accidentally start leaking api_tokens everywhere